ramls/schema/loan-policy-id.json

```diff
@@ -7,6 +7,24 @@
     "loanPolicyId": {
       "description": "ID of the loan policy determined by the circulation rules",
       "type": "string"
+    },
+    "appliedRuleConditions": {
```

Comment:
I think now that we've had clarification from Cheryl, this can be modelled using the domain language.

My interpretation of that, is that this would be an `applicableRule` property, and that would have properties for the relevant parts of the rule (e.g. `materialTypeId`, `loanTypeId`, `patronGroupId` etc).

I know this has had a lot of back and forth, which is likely frustrating. What do you think of this modelling?

Reply:
> and that would have properties for the relevant parts of the rule (e.g. materialTypeId, loanTypeId, patronGroupId etc).

Do you suggest using ids instead of boolean fields?

Reply:
@roman-barannyk  I think that @julianladisch comments on the issue suggests that rules can be more complicated than this and we should not try to change this API design until we have clarified what the intended behaviour is when either the `all` keyword is used or multiple records are included in a single rule.